docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
 
 *   Upload edits to Wikidata
 *   Export to QuickStatement
 *   Export Wikidata schema
 
+### Custom tabular exporter
+
+![A screenshot of the custom tabular content tab.](/img/custom-tabular-exporter.png)
+
+With the custom tabular exporter, you can choose which of your data to export, the separator you wish to use, and whether you'd like to download it to your computer or upload it into a Google Sheet. 
+
+You can drag and drop the columns appearing in the column list to reorder the output. The options for reconciled and date data are applied to each column individually. 
+
+This exporter is especially useful with reconciled data, as you can choose whether you wish to output the cells' original values, the matched values, or the matched IDs. Ouputting “match entity's name”, “matched entity's ID”, or “cell's content” will output, respectively, the contents of `cell.recon.match.name`, `cell.recon.match.id`, and `cell.value`. 
+
+“Output nothing for unmatched cells” will export empty cells for both newly-created matches and cells with no chosen matches. “Link to matched entity's page” will produce hyperlinked text in an HTML table output, but have no change in other formats.
+
+You can also choose how to [output date-formatted cells](exploring#dates). You can create a custom date output by using [formatting according to the SimpleDateFormat parsing key found here](grelfunctions#todateo-b-monthfirst-s-format1-s-format2-).
+
+![A screenshot of the custom tabular file download tab.](/img/custom-tabular-exporter2.png)
+
+With the download options, you can generate a preview of how the first ten rows of your dataset will output. If you do not choose one of the file formats on the right, the "Download" button will generate a text file. 
+
+With the <span class="menuItems">Option Code</span> tab, you can copy JSON data that saves your current settings, to reuse on another project, or you can paste in existing JSON settings data to apply to the current project. 
+
+### SQL exporter
+
+The SQL exporter creates a SQL statement containing the data you’ve exported, which you can use to overwrite or add to an existing database. 
+
+![A screenshot of the SQL statement creator window.](/img/sql-exporter.png)
+
+A window with two tabs will pop up: one to define what data to output, and another to modify the statement table. This allows you to craft a complete SQL command, including DROP and IF EXISTS if you require them. 
+
+You can set a default value for empty cells after unchecking “Allow null” in one or more columns. Trimming column names will remove their whitespace characters. 
+
+![A screenshot of the SQL statement download window.](/img/sql-exporter2.png)
+
+“Include schema” means that you will start your statement with the creation of a table. Without that, you will only have an INSERT statement. 
```

Comment:
I feel this would be easier to read with some more structured layout - maybe a table? I also feel we should probably list all the options together (at the moment some options are explained above the graphic and some under the graphic - I think we should keep them all together

Reply:
Now that I think of it, the reasons why some thing are included on "Content" and some on "Download" are not clearly defined to me. The options on the "Download" tab include the option to exclude everything you decided on the "Content" tab; to me, it makes sense to present those checkboxes at the outset.
I wonder if these options should be moved around, or if I'm just missing why someone would want the workflow to be this way.

Reply:
I think this is the only unresolved bit on this PR. If everyone could give this section a re-read, I can use some guidance on a good way to present this information. 

https://deploy-preview-3336--openrefine-docs.netlify.app/manual/exporting/#sql-exporter